airflow/providers/databricks/hooks/databricks.py

```diff
@@ -135,9 +137,27 @@ def __init__(
     ) -> None:
         super().__init__(databricks_conn_id, timeout_seconds, retry_limit, retry_delay, retry_args, caller)
 
+    def create(self, json: dict) -> int:
+        """
+        Utility function to call the ``api/2.1/jobs/create`` endpoint.
+
+        :param json: The data used in the body of the request to the ``create`` endpoint.
+        :return: the job_id as an int
```

Comment:
I know this is not consistent i.e at some places we have return in a docstring but someplace not. But can we keep consistent in this PR? Thank you!

Reply:
hey @pankajastro this has a return for the job id, are you saying to put a docstring for when the return for a method is None? Such as the reset method?

Reply:
you are right, the other method returns None.